templates/tools/dockerfile/test/python_phusion_3.6_x64/Dockerfile.template

```diff
@@ -13,8 +13,14 @@
   # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   # See the License for the specific language governing permissions and
   # limitations under the License.
+  FROM phusion/baseimage:0.11
```

Comment:
Is `phusion` image releases generally stable? I'm fine pining it since it haven't reach `1.0`.

Reply:
What do you mean by "stable"?

Reply:
I mean should we unpin it and use the latest `phusion` image?